lib/options/options.js

```diff
@@ -1,44 +1,44 @@
 export const options = [
   {
+    type: 'password',
+    name: 'githubToken',
+    label: 'Access token',
+    sensitiveData: true,
+    description: 'If you want better <strong>Sass, Less or Haskell support</strong> for private repositories, you\'ll need to <a href="https://github.com/settings/tokens/new?scopes=repo&description=OctoLinker%20browser%20extension">create a token</a> with the repo permissions.',
+  },
+  {
     type: 'checkbox',
     name: 'newWindow',
-    label: 'Open link in new tab',
-    value: undefined,
+    label: 'New tab',
+    description: 'Open link in a new tab.',
     defaultValue: false,
   },
   {
     type: 'checkbox',
     name: 'newWindowActive',
-    label: 'Focus new tabs',
-    value: undefined,
+    label: 'Focus new tab',
+    description: 'Focus new tab when opening a link.',
     defaultValue: true,
   },
   {
     type: 'checkbox',
     name: 'showLinkIndicator',
-    label: 'Show indicator if line contains OctoLinker links',
-    value: undefined,
-    defaultValue: true,
-  },
-  {
-    type: 'checkbox',
-    name: 'doTrack',
-    label: 'Send anonymous usage reports to Google Analytics',
-    value: undefined,
+    label: 'Line indicator',
+    description: 'Show an indicator if line contains OctoLinker links.',
     defaultValue: true,
   },
   {
     type: 'checkbox',
     name: 'showUpdateNotification',
-    label: 'Show update notification',
-    value: undefined,
+    label: 'Update notification',
+    description: 'Show a notification if a new version is available.',
     defaultValue: true,
   },
   {
     type: 'checkbox',
-    name: 'debugMode',
-    label: 'Debug mode',
-    value: undefined,
-    defaultValue: false,
+    name: 'doTrack',
+    label: 'Usage reports',
+    description: 'Send anonymous usage reports to Google Analytics.',
+    defaultValue: true,
```

Comment:
No, it was on purpose. I think the tracking options is less important and a lot of other apps are  putting this option to the end as well.